Document WEAVE_MTU in weave-kube config options

awh opened the pull request.

awh commented:
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/kube-addon/)

rade commented on site/kube-addon.md:
```diff
@@ -144,3 +144,7 @@ The list of variables you can set is:
+* WEAVE\_MTU - Weave Net defaults to 1410 bytes, but you can set a
+  smaller size if your underlying network has a tighter limit, or set
+  a larger size for better performance if your network supports jumbo
+  frames.
```
This should probably link to https://www.weave.works/docs/net/latest/using-weave/fastdp/ (ideally the sub-section that talks about MTUs, but it looks like we don't support sub-section linking, grrr).

bboreham replied:
Sub-sections can be linked to if an anchor is created, e.g. like https://www.weave.works/docs/net/latest/using-weave/service-management/#routing

awh replied:
> it looks like we don't support sub-section linking, grrr

PRs gratefully received at weaveworks/wordepress :smile:

awh commented:
#### Documentation changes:
modified [kube-addon](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/kube-addon/)
modified [using-weave/fastdp](https://dev-weavewww.pantheonsite.io/docs/net/document-mtu-with-weave-kube/using-weave/fastdp/)
